src/app/app.component.html

```diff
@@ -1,21 +1,12 @@
-<!--The content below is only a placeholder and can be replaced.-->
-<div style="text-align:center">
-	<h1>
-		Welcome to {{ title }}!
-	</h1>
-	<img width="300" alt="Angular Logo" src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNTAgMjUwIj4KICAgIDxwYXRoIGZpbGw9IiNERDAwMzEiIGQ9Ik0xMjUgMzBMMzEuOSA2My4ybDE0LjIgMTIzLjFMMTI1IDIzMGw3OC45LTQzLjcgMTQuMi0xMjMuMXoiIC8+CiAgICA8cGF0aCBmaWxsPSIjQzMwMDJGIiBkPSJNMTI1IDMwdjIyLjItLjFWMjMwbDc4LjktNDMuNyAxNC4yLTEyMy4xTDEyNSAzMHoiIC8+CiAgICA8cGF0aCAgZmlsbD0iI0ZGRkZGRiIgZD0iTTEyNSA1Mi4xTDY2LjggMTgyLjZoMjEuN2wxMS43LTI5LjJoNDkuNGwxMS43IDI5LjJIMTgzTDEyNSA1Mi4xem0xNyA4My4zaC0zNGwxNy00MC45IDE3IDQwLjl6IiAvPgogIDwvc3ZnPg==">
-</div>
-<h2>Here are some links to help you start: </h2>
-<ul>
-	<li>
-		<h2><a target="_blank" rel="noopener" href="https://angular.io/tutorial">Tour of Heroes</a></h2>
-	</li>
-	<li>
-		<h2><a target="_blank" rel="noopener" href="https://github.com/angular/angular-cli/wiki">CLI Documentation</a></h2>
-	</li>
-	<li>
-		<h2><a target="_blank" rel="noopener" href="https://blog.angular.io/">Angular blog</a></h2>
-	</li>
-</ul>
-
-<router-outlet></router-outlet>
+<mat-toolbar class="mat-elevation-z5 primary-nav" role="heading" color="primary">
+	<a mat-button class="title" routerLink="/">{{ title }}</a>
+	<div fxFlex></div>
+	<a mat-icon-button href="{{ githubUrl }}">
+		<mat-icon [attr.aria-label]="title + 'GitHub'" svgIcon="github"></mat-icon>
```

Comment:
Space is needed between title and GitHub
```suggestion
		<mat-icon [attr.aria-label]="title + ' GitHub'" svgIcon="github"></mat-icon>
```